src/js/video.js

```diff
@@ -566,5 +567,7 @@ videojs.dom = Dom;
  */
 videojs.url = Url;
 
+videojs.defineLazyProperty = defineLazyProperty;
```

Comment:
any reason to expose it currently?

Reply:
I think we will need it in a few other places, vhs for sure.